Userland/Libraries/LibRegex/RegexByteCode.cpp

```diff
@@ -659,6 +659,18 @@ ALWAYS_INLINE bool OpCode_Compare::compare_string(MatchInput const& input, Match
 
 ALWAYS_INLINE void OpCode_Compare::compare_character_class(MatchInput const& input, MatchState& state, CharClass character_class, u32 ch, bool inverse, bool& inverse_matched)
 {
+    auto is_space_or_line_terminator = [](u32 code_point) {
```

Comment:
Should this only be done in unicode mode, or non-unicode mode as well?

Reply:
I think both modes because the grammar isn't distinguished with `[u]` (unicode) or `[~u]` (not-unicode) here.

Reply:
Ah alright, then 👍